atf_eregs/static/regulations/css/less/module/custom/full-footer.less

```diff
@@ -33,6 +33,14 @@
 		ul li:not(:first-child) {
 			text-align: right;
 		}
+        .footer-small ul li {
+			width: 20%;
+            white-space: nowrap;
+            text-size: 12px;
+        }
+        .footer-small ul li a.external{
+            font-size: 12px;
+        }
```

Comment:
I thought this only affected wide screens, i.e. screens at least 480px wide. That was my intent, because I think the footers will wrap correctly on smaller screens. I didn't test that, though…